### ISSUE TITLE
[A11y] - Focused element is completely covered by other content

### ISSUE THREAD

**Issue by VaniChinnam** [dequelabs/cauldron#1398]
### Current Behaviour 
`The focused element is completely covered by other author-created content. Element description 1: "Still Yes" button.
Element location: In Example, under "Do you like coffee?".
Description of the issue: The Focus element is completely covered while tabbing through the "tab" key. Focus is not at all visible to the user. Element Description 2: "Source" link and "Still Yes" radio button.
Element location: In the "Radio card group". Note: The above-mentioned elements are  completely covered by the header section on the "Shift- tab" focus.`<br/><hr/>
### Expected Behaviour 
`RULE :  When a user interface component receives keyboard focus, the component MUST NOT be entirely hidden due to author-created content. HOW TO FIX:  Fix this issue by using an applicable technique below: For content such as sticky headers, sticky footers, cookie consent banners, etc: 1. Use a technique such as CSS scroll-padding to ensure sticky headers or sticky footers do not obscure the focused item when tabbing forward or backward through the page. For overlays such as non-modal dialogs, tooltips, submenus, chat widgets, message panels, etc. that are opened and closed BY THE USER (e.g. by using the Spacebar or Enter key or when tabbed to): 1. Position the popup so it does not cover other focusable content OR  2. Make the content modal and move focus to it and trap focus in it OR  3. Close the content when focus leaves it OR  4. Provide a way for the user to close the overlay content (e.g. the ESC key) while focus is on an item behind the overlay. For overlays such as non-modal dialogs, popups, submenus, chat widgets, message panels, etc. that open AUTOMATICALLY: 1. Position the popup so it does not cover other focusable content OR  2. Make the content modal and move focus to it and trap focus in it OR  3. Don’t open the content automatically. Allow the user to open and close it and follow the recommendations for user-controlled content above. For content such as modal dialogs that don’t automatically get and trap focus when opened: 1. Move focus to the modal dialog and trap focus inside it. `<br/><hr/>
### Impact 
 <p>

- [ ] Blocker
- [ ] Serious
- [ ] Critical
- [x] Moderate
- [ ] Minor

### Issue Details

<details><summary>View Issue 1625838 </summary> 
 <p>

_**Auditor ID**_ : [1625838](https://axeauditor.dequecloud.com/issue-global-search?searchKeyword=1625838&searchOn=id) 

_**description**_ : undefined 

_**Page Name**_ : Radio card group 

_**source**_ : `<div class="Panel__Content"><div class="RadioCardGroup__Checked"></div><div class="RadioCardGroup__Base"><div class="RadioCardGroup__Image"><div style="background-color: green; height: 60px; width: 60px; border-radius: 60px;"></div></div><label for="still-yes" class="RadioCardGroup__Label">Still Yes</label></div></div>` 

_**Screenshots**_ : [Focused element is completely covered issue](https://axeauditor.dequecloud.com/api/v1/file/44c9ee1c-be74-11ee-beea-7b4096c34f98
Focused element is completely covered issue: https://axeauditor.dequecloud.com/api/v1/file/44ca6090-be74-11ee-a8e8-3be7d3a9bd1a
Still Yes: https://axeauditor.dequecloud.com/api/v1/file/69b290f4-bf8b-11ee-91d6-f7e75daa30bd
Source: https://axeauditor.dequecloud.com/api/v1/file/69b2b430-bf8b-11ee-ada9-bb9e62d69fec) 

</p>
</details>

**Comment by scurker** [comment]
@VaniChinnam I'm having a little trouble following exactly what the issue is here. What element is focus being obscured on? I tested the page myself and all of the focusable elements appear to have focus, so maybe I'm missing something.

**Comment by VaniChinnam** [comment]
@scurker The elements are receiving focus and have visible focus but the elements are not visible to the users when focused because of the sticky header/ sticky footer.
Attached the NVDA speech viewer.

Still yes button during shift+tab navigation - Not visible due to sticky header
![image](https://github.com/dequelabs/cauldron/assets/47885015/5ee03dd2-35c7-4b2d-9e5b-373c64697663)

Source link during shift+tab navigation - Not visible due to sticky header
![image](https://github.com/dequelabs/cauldron/assets/47885015/60f70a8f-bd1e-4e36-b6d2-8fd17d74bcc8)

Still Yes button during tab key navigation - Not visible due to 
![image](https://github.com/dequelabs/cauldron/assets/47885015/69c96f5d-caad-4017-9c9d-1c34862bcc67)

**Comment by scurker** [comment]
This component is deprecated and is slated for removal. #1289 will be the eventual replacement for this component.
